installer/scripts/tomlparser.rb

```diff
@@ -16,13 +18,13 @@
 @excludePath = "*.csv2" #some invalid path
 
 # Use parser to parse the configmap toml file to a ruby structure
-def parseConfigMap
+def parseConfigMap(path)
   begin
     # Check to see if config map is created
-    if (File.file?(@configMapMountPath))
-      puts "config::configmap container-azm-ms-agentconfig for settings mounted, parsing values"
-      parsedConfig = Tomlrb.load_file(@configMapMountPath, symbolize_keys: true)
-      puts "config::Successfully parsed mounted config map"
+    if (File.file?(path))
+      puts "config::configmap container-azm-ms-agentconfig for settings mounted, parsing values from #{path}"
+      parsedConfig = Tomlrb.load_file(path, symbolize_keys: true)
+      puts "config::Successfully parsed mounted config map from #{path}"
       return parsedConfig
     else
       puts "config::configmap container-azm-ms-agentconfig for settings not mounted, using defaults"
```

Comment:
Can we add which is not mounted in the 'else' path, now that we use the same for 2 files ?

Reply:
Done